src/hacbs/components/Commits/CommitsListRow.tsx

```diff
@@ -0,0 +1,73 @@
+import * as React from 'react';
+import { Link } from 'react-router-dom';
+import { GithubIcon } from '@patternfly/react-icons/dist/js/icons/github-icon';
+import ActionMenu from '../../../shared/components/action-menu/ActionMenu';
+import ExternalLink from '../../../shared/components/links/ExternalLink';
+import { RowFunctionArgs, TableData } from '../../../shared/components/table';
+import { Timestamp } from '../../../shared/components/timestamp/Timestamp';
+import { Commit } from '../../types';
+import { useCommitActions } from './commit-actions';
+import { commitsTableColumnClasses } from './CommitsListHeader';
+
+const CommitsListRow: React.FC<RowFunctionArgs<Commit>> = ({ obj }) => {
+  const actions = useCommitActions(obj);
+  return (
+    <>
+      <TableData className={commitsTableColumnClasses.name}>
+        <Link to={`/app-studio/commit?name=${obj.sha}`} title={obj.sha} className="pf-u-mr-sm">
+          {obj.sha.length > 14 ? `${obj.sha.slice(0, 7)}` : obj.sha}
```

Comment:
sha is 40 characters long, so no need to have a check here.

```suggestion
          {`${obj.sha.slice(0, 7)}`}
```

Reply:
updated